streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableFilter.java

```diff
@@ -77,6 +77,7 @@ public void process(K key, Change<V> change) {
             V newValue = computeValue(key, change.newValue);
```

Comment:
Actually never mind, I was originally thinking that after this patch we should `if (change.oldValue == null && change.newValue == null) throw StreamsException`, since it should not happen; but I just realized a mapValues operator can still make both values to be null and pass it to the downstream filter.